.github/workflows/cypress.yaml

```diff
@@ -181,30 +165,6 @@ jobs:
                   # make Node's os.tmpdir() return something where we actually have permissions
                   TMPDIR: ${{ runner.temp }}
 
-                  # tell Cypress more details about the context of this run
-                  COMMIT_INFO_BRANCH: ${{ github.event.workflow_run.head_branch }}
-                  COMMIT_INFO_SHA: ${{ github.event.workflow_run.head_sha }}
-                  COMMIT_INFO_REMOTE: ${{ github.repositoryUrl }}
-                  COMMIT_INFO_MESSAGE: ${{ needs.prepare.outputs.commit_message }}
-                  COMMIT_INFO_AUTHOR: ${{ needs.prepare.outputs.commit_author }}
-                  COMMIT_INFO_EMAIL: ${{ needs.prepare.outputs.commit_email }}
-                  CYPRESS_PULL_REQUEST_ID: ${{ needs.prepare.outputs.pr_id }}
-                  CYPRESS_PULL_REQUEST_URL: https://github.com/${{ github.repository }}/pull/${{ needs.prepare.outputs.pr_id }}
-
-                  # pass the Percy token as an environment variable
-                  PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
-
-                  # only run percy on legacy crypto (for now)
-                  PERCY_ENABLE: ${{ matrix.crypto == 'legacy' && needs.prepare.outputs.percy_enable || 0 }}
-                  PERCY_BROWSER_EXECUTABLE: ${{ steps.setup-chrome.outputs.chrome-path }}
-                  # tell Percy more details about the context of this run
-                  PERCY_BRANCH: ${{ github.event.workflow_run.head_branch }}
-                  PERCY_COMMIT: ${{ github.event.workflow_run.head_sha }}
-                  PERCY_PULL_REQUEST: ${{ needs.prepare.outputs.pr_id }}
-                  PERCY_PARALLEL_NONCE: ${{ needs.prepare.outputs.uuid }}
-                  # We manually finalize the build in the report stage
-                  PERCY_PARALLEL_TOTAL: -1
```

Comment:
Given you're removing this we should probably remove all other instances of Percy in this workflow to avoid confusion when someone tries to turn it back on and it doesn't work

Reply:
Ah didn't mean to remove the percy config, we'll probably need it when we turn percy back on no?

Reply:
Yes, feel free to add it back